Overview.src.html

```diff
@@ -3406,9 +3406,9 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      <dd><var>max-age</var>
 
      <dt><span title=concept-cache-credentials>credentials</span>
-     <dd>False if <var>request</var>'s
-     <span title=concept-request-credentials-mode>credentials mode</span> is not
-     "<code title>include</code>", and true otherwise
+     <dd>True if <var>request</var>'s
+     <span title=concept-request-credentials-mode>credentials mode</span>
+     is "<code title>include</code>", false otherwise
```

Comment:
Please keep "and" here and below.

And feel free to add your name to the list of Acknowledgements.